src/test/resources/samples/bibframe/authority_person.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "123456654322",
+  "label": "person-label",
+  "type": "PERSON",
+  "identifiers": [
+    {
+      "value": "2024-202454/PR/r006",
```

Comment:
For PERSON, let us use the LCCN "n 1234567890" (note that there is a space after "n". This should get removed by the DefaultLccnNormalizer)